lib/group_email_credentials_check.rb

```diff
@@ -7,6 +7,9 @@
 # problem checks, and if any credentials have issues they will show up on
 # the admin dashboard as a high priority issue.
 class GroupEmailCredentialsCheck
+  def self.retries = 2
```

Comment:
I do wonder if `max_retries` might be a better name to signal that this is a limit, and not a count we are keeping track of?

Reply:
Agree!